_topics/en/vaults.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Vaults
```

Comment:
By the way there was a discussion today about this,
https://diyhpl.us/wiki/transcripts/london-bitcoin-devs/2020-05-19-socratic-seminar-vaults/

There's also Jeremy's vault implementation which is in his bip119 branch.

There's also Fidelity's version- https://github.com/fmr-llc/Vault-mbed

and this thing https://github.com/JSwambo/bitcoin-vault